portfolio/src/main/java/com/google/sps/Events.java

```diff
@@ -0,0 +1,138 @@
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import java.util.Arrays;
+
+public final class Events {
+  private static final String PERSON_AMELIA = "Amelia";
+  private static final String PERSON_AVA = "Ava";
+  private static final String PERSON_EMMA = "Emma";
+  private static final String PERSON_ISABELLA = "Isabella";
+  private static final String PERSON_JAMES = "James";
+  private static final String PERSON_LIAM = "Liam";
+  private static final String PERSON_LOGAN = "Logan";
+  private static final String PERSON_NOAH = "Noah";
+  private static final String PERSON_OLIVER = "Oliver";
+  private static final String PERSON_OLIVIA = "Olivia";
+
+  private static final int TIME_0800 = TimeRange.getTimeInMinutes(8, 0);
+  private static final int TIME_0830 = TimeRange.getTimeInMinutes(8, 30);
+  private static final int TIME_0900 = TimeRange.getTimeInMinutes(9, 0);
+  private static final int TIME_0930 = TimeRange.getTimeInMinutes(9, 30);
+  private static final int TIME_1000 = TimeRange.getTimeInMinutes(10, 0);
+  private static final int TIME_1030 = TimeRange.getTimeInMinutes(10, 30);
+  private static final int TIME_1100 = TimeRange.getTimeInMinutes(11, 0);
+  private static final int TIME_1130 = TimeRange.getTimeInMinutes(11, 30);
+  private static final int TIME_1200 = TimeRange.getTimeInMinutes(12, 0);
+  private static final int TIME_1230 = TimeRange.getTimeInMinutes(12, 30);
+  private static final int TIME_1300 = TimeRange.getTimeInMinutes(13, 0);
+  private static final int TIME_1330 = TimeRange.getTimeInMinutes(13, 30);
+  private static final int TIME_1400 = TimeRange.getTimeInMinutes(14, 0);
+  private static final int TIME_1430 = TimeRange.getTimeInMinutes(14, 30);
+  private static final int TIME_1500 = TimeRange.getTimeInMinutes(15, 0);
+  private static final int TIME_1530 = TimeRange.getTimeInMinutes(15, 30);
+  private static final int TIME_1600 = TimeRange.getTimeInMinutes(16, 0);
+  private static final int TIME_1630 = TimeRange.getTimeInMinutes(16, 30);
+  private static final int TIME_1700 = TimeRange.getTimeInMinutes(17, 0);
+  private static final int TIME_1730 = TimeRange.getTimeInMinutes(17, 30);
+  private static final int TIME_1800 = TimeRange.getTimeInMinutes(18, 0);
+  private static final int TIME_1830 = TimeRange.getTimeInMinutes(18, 30);
+  private static final int TIME_1900 = TimeRange.getTimeInMinutes(19, 0);
+  private static final int TIME_1930 = TimeRange.getTimeInMinutes(19, 30);
+  private static final int TIME_2000 = TimeRange.getTimeInMinutes(20, 0);
+  private static final int TIME_2030 = TimeRange.getTimeInMinutes(20, 30);
+  private static final int TIME_2100 = TimeRange.getTimeInMinutes(21, 0);
+
+  public static final Event[] events = {
```

Comment:
I assume this was provided code, but I'll just note that in most circumstances you wouldn't hard-code this kind of thing as a class. Instead you'd be loading it from somewhere like a file, or database, or some remote API call, depending on what you're building. (Obviously don't bother doing that here, just wanted to point it out.)